draft-ietf-teas-network-slice-topology-yang.txt

```diff
@@ -1558,23 +1559,38 @@ A.1.  Consideration on Reusing ACTN VN for Network Slicing
    additionally, would bind the network slice solution to TE
    technologies.
 
+   The proposed models in this draft aim to deliver a solution
+   equivalent to Type 2 VN to provide optional path constraints and
+   topology intent within the context of RFC 9543 network slicing.
 
 
 
+Liu, et al.              Expires 6 October 2025                [Page 28]
+
+Internet-Draft      Network Slice Topology Data Model         April 2025
 
 
+   These models complement the existing solution outlined in
+   [I-D.ietf-teas-ietf-network-slice-nbi-yang], while ensuring
+   consistent use of SLO and SLE attributes in a technology-agnostic
+   manner to express customer intent.
 
+   In a nutshell:
 
-Liu, et al.             Expires 18 September 2025              [Page 28]
-
-Internet-Draft      Network Slice Topology Data Model         March 2025
+   *  the data models, defined in this draft, are intended to be used
+      when there is a need to extend the connectivity intent expressed
+      using the Network Slice Service data model defined in
+      [I-D.ietf-teas-ietf-network-slice-nbi-yang];
 
+   *  the VN type 2 data models, defined in [RFC9731], are intended to
```

Comment:
"when there is a need to extend,  with a major control over network resources allocation by the customer,  the connectivity intent...."

Reply:
Addressed with minor rephrasing in 51119ea27bdaae1fed67564e58bcd4ce9b138005